src/index.html

```diff
@@ -19,10 +19,15 @@
             </nav>
         </div>
     </header>
-    <main id="movies-container">
-        
+    <main>
+       <section id="movies-container" class="show">
+
+       </section>
+       <section id="popup-window" class="hide">
+
+       </section>
```

Comment:
I think it would look better if popup window has different background color than the body background color to give more contrast.